packages/legacy/core/App/screens/PINExplainer.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react'
+/*
+import { useNavigation } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { AuthenticateStackParams, Screens } from '../../types/navigators'
+*/
+import { Trans, useTranslation } from 'react-i18next'
+import Button, { ButtonType } from '../components/buttons/Button'
+import BulletPoint from '../components/inputs/BulletPoint'
+import { testIdWithKey } from '../utils/testable'
+
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+
+import { useTheme } from '../contexts/theme'
+
+export interface PINExplainerProps {
+  continueCreatePIN: () => void
+}
+
+const Component: React.FC<PINExplainerProps> = ({ continueCreatePIN }) => {
```

Comment:
```suggestion
const PINExplainer: React.FC<PINExplainerProps> = ({ continueCreatePIN }) => {
```
Component should be named even though it's a default export, it's the existing pattern and it helps readability.

Reply:
Agree